Gems/DiffuseProbeGrid/Code/Source/Render/DiffuseProbeGridFeatureProcessor.cpp

```diff
@@ -125,7 +125,7 @@ namespace AZ
                 // load probe visualization model, the BLAS will be created in OnAssetReady()
                 m_visualizationModelAsset = AZ::RPI::AssetUtils::GetAssetByProductPath<AZ::RPI::ModelAsset>(
                     "Models/DiffuseProbeSphere.azmodel",
-                    AZ::RPI::AssetUtils::TraceLevel::Assert);
+                    AZ::RPI::AssetUtils::TraceLevel::Warning);
```

Comment:
I'm actually wondering if it should even be a warning here? Isn't this model only used for debug visualization? If so we could move the warning to the point where the user turns on that debug visualization.  Also I'm hoping this model isn't required for a release build.

Reply:
Proper fix should be to not render visualization model if the asset is not loaded and just wait for the asset to be loaded.

Reply:
I'll file a GHI for the after this release, as the proper fix wouldn't meet the current bar.

Reply:
Filed https://github.com/o3de/o3de/issues/15819